### ISSUE TITLE
Regarding filenames.pickle and class_info.pickle

### ISSUE THREAD

**Issue by saba56** [chenqi008/BoGAN#4]
The above two files are not being dumped anywhere....How to generate those two files.

**Comment by chenqi008** [comment]
Thanks to point out that.
To make sure that the filenames of caption and video are the same, we reorganize the dataset in the released version, and also split the training set and test set into different folders directly. Thus, the files filenames.pickle, containing all the filenames, is not necessary anymore. Maybe you can try to refine the dataset.py to suit the new dataset construction.
Hope helpful.
